### ISSUE TITLE
AngelList Changed the Name

### ISSUE THREAD

**Issue by conceptminers1** [goldenrecursion/predicates#84]
AngelList is now named Wellfound. So does this triple still needs verification?

**Comment by jedchristiansen** [comment]
Yes, these triples still require verification.  The existing URLs still work.  While a new brand has been announced, the future URL structure hasn't been announced, so we're waiting on that before scoping any additional work.

**Comment by conceptminers1** [comment]
Thanks Jed. Have a great day!



Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________
